docs/topics/proc_show-debug-logging-on-openshift-vertx.adoc

```diff
@@ -0,0 +1,89 @@
+
+[#accessing-debug-log-output-on-openshift_{context}]
+= Accessing Debug Log Output on OpenShift
+
+This procedure shows you how to view the debug log output of your Java application running in a container on OpenShift.
+
+.Prerequisites
+* An application with debug logging runnig on OpenShift.
+* The `oc` client authenticated.
+
+.Procedure
+
+. Using the `oc` command, list the available deployment configurations:
++
+[source,bash]
+----
+$ oc get dc
+----
+
+// does the same env var apply for Vert.x?
+. Set the `JAVA_ENABLE_DEBUG` environment variable in the deployment configuration of your application to `true`:
++
+[source,bash,subs="attributes+"]
+----
+$ oc set env dc/{app-name} JAVA_ENABLE_DEBUG=true
+----
++
+. If configured to do so, you application will redeploy after the configuration change.
+To manually redeploy your application, execute:
++
+[source,bash,subs="attributes+"]
+----
+$ oc rollout latest dc/{app-name}
+----
++
+. Obtain the name of your pod:
++
+[source,bash,options="nowrap",subs="attributes+"]
+----
+$ oc get pods
+----
++
+// Add alternative workflow using web console?
+. Follow the console log output for the pod you have just redeployed:
++
+----
+$ oc logs -f pod/{app-name}-2-aaaaa
+----
++
+. Keep the terminal window displaying the log output open, so you can continue following the log output.
++
+. Interact with your application.
++
+For example, if you had debug logging in the xref:mission-http-api-vertx[{name-mission-http-api} booster] to log the `name` variable in the `/api/greeting` method:
```

Comment:
can you update the xref target to use {context} or some other var so it can be used across other runtimes?